cmd/podman/create.go

```diff
@@ -626,19 +632,37 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime, imageName string,
 		return nil, errors.Errorf("cannot pass additional search domains when also specifying '.'")
 	}
 
+	ImageVolumes := []string{}
+	for k := range data.ContainerConfig.Volumes {
+		ImageVolumes = append(ImageVolumes, k)
```

Comment:
Can't you just do 
imageVolumes := data.ContainerConfig.Volumes

Reply:
fixed